### PR TITLE
Enable import/order rule

### PR DESCRIPTION
**Pull request by petrhanak**
Now that it's fixable, there is no reason to not have it

### CODE REVIEW

File: environments/shared/recommended.js [hunk 1]
```diff
@@ -663,6 +663,12 @@ module.exports = {
 
     // Forbid a module from importing itself
     'import/no-self-import': 'error',
+
+    // Enforce a convention in the order of require() / import statements
+    'import/order': ['error', {
```
Comment (robertrossmann) [inline environments/shared/recommended.js]: This is stylistic rule and therefore must not cause build failures (randomly-ordered imports do not imply broken code).

Please change to `warn` and move to `coding-styles/fixable`, thanks! 🙏

File: environments/shared/recommended.js [hunk 2]
```diff
@@ -663,6 +663,12 @@ module.exports = {
...
+      // todo: enable moment newslines-between when it will be fixable
```
Comment (robertrossmann) [inline environments/shared/recommended.js]: Please make this a `@TODO`. 😄

Also I am not in agreement with having newlines between the groups; I'd rather not have newlines between groups at all or not enforce any particular style. It just looks weird if you have only one import of a particular type in a module and there are more types used, ie.

```js
import fs from 'fs'

import _ from 'lodash'

import Component from './component')
```

So many unnecessary spaces... 🤔

Comment (petrhanak) [reply environments/shared/recommended.js]: good point, removing then

**Review by robertrossmann — APPROVED** [review]
🍻

**Pull request merged**